src/app/content/studyGuides/selectors.ts

```diff
@@ -81,21 +88,57 @@ export const loadedCountsPerSource = createSelector(
   getLoadedCountsPerSource
 );
 
+export const highlightColorFiltersWithContent = createSelector(
+  totalCountsPerPageOrEmpty,
+  getHighlightColorFiltersWithContent
+);
+
+export const defaultLocationFilter = createSelector(
+  studyGuidesLocationFilters,
+  parentSelectors.page,
+  (filters, currentPage) => currentPage && getHighlightLocationFilterForPage(filters, currentPage)
+);
+
+const filtersFromQuery = createSelector(
+  navigationSelectors.query,
+  (query) => getFiltersFromQuery(query)
+);
+
+export const unloggedAndQueryMissingFirstChapter = createSelector(
+  authSelectors.loggedOut,
+  parentSelectors.firstChapter,
+  filtersFromQuery,
+  (notLoggedIn, firstChapter, queryFilters) =>
+    notLoggedIn && firstChapter && !queryFilters.locationIds.includes(firstChapter.id)
+);
+
+export const loggedAndQueryMissingLocationIds = createSelector(
+  authSelectors.loggedOut,
+  filtersFromQuery,
+  defaultLocationFilter,
+  (notLoggedIn, queryFilters, defaultFilter) =>
+    !notLoggedIn && queryFilters.locationIds.length === 0 && defaultFilter
+);
+
 export const summaryFilters = createSelector(
   localState,
-  (state) => state.summary.filters
+  unloggedAndQueryMissingFirstChapter,
+  loggedAndQueryMissingLocationIds,
+  parentSelectors.firstChapter,
+  highlightColorFiltersWithContent,
+  defaultLocationFilter,
+  (state, unlogged, logged, firstChapter, colorsWithSG, defaultFilter) => unlogged && firstChapter
+    ? { colors: Array.from(colorsWithSG.size ? colorsWithSG : colorfilterLabels), locationIds: [firstChapter.id] }
+    : (logged && defaultFilter
+      ? { colors: Array.from(colorsWithSG.size ? colorsWithSG : colorfilterLabels), locationIds: [defaultFilter.id] }
+      : state.summary.filters)
```

Comment:
you're not using the user selected filters ever if the user is logged in ?

i think this should be first and all the other logic only happen if its null/empty right?

could you simplify logic by combining into a "defaultFIlters" selector that combines both color and location, doing your auth check in there since the default filters are different if you're logged in or not, and then in summary filters basically just go `=> ({...defaultFilters, ...state.summary.filters})` (btw you should select `state.summary.filters` in another selector so that you aren't re-generating this filters value whenever the whole SG state changes)